presto-hive/src/main/java/com/facebook/presto/hive/SseType.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive;
+
+public enum SseType
+{
+    KMS("KMS"),
```

Comment:
This can be a simple enum:
```java
public enum SseType
{
    S3,
    KMS;
}
```
I noticed that `PrestoS3SignerType` could also be simplified: https://github.com/prestodb/presto/pull/7295